data/human/hails.txt

```diff
@@ -2708,6 +2708,195 @@ phrase "friendly civilian"
 		"neodymium and neodymium" "neodymium"
 		"neodymium or neodymium" "neodymium"
 
+phrase "friendly civilian"
+	word
+		"Be careful of"
+		"A warning about"
+		"Be suspicious of"
+		"Don't trust"
+	word
+		" "
+	word
+		"those space casinos. I blew"
+	word
+		" "
+	word
+		"half a"
+		"a"
+		"two"
+		"three"
+		"four"
+		"five"
+	word
+		" million credits on "
+	word
+		"Martini"
+		"Thrall"
+		"Geminus"
+		"Hestia"
+		"Vinci"
+		"Vail"
+		"Calda"
+		"Pearl"
+		"Glory"
+		"Farseer"
+		"Pilot"
+		"Ada"
+		"Hermes"
+		"Mainsail"
```

Comment:
I'm not sure Pilot makes sense on this list.
https://github.com/endless-sky/endless-sky/blob/02e6ded1e7cb9ef7a8e401e71c9accd6133a68b5/data/map%20planets.txt#L3477-L3482
Doesn't appear to be the sort of place where you're likely to find a casino, at least not the sort that these other worlds would have.

Thrall is also a questionable choice:
https://github.com/endless-sky/endless-sky/blob/02e6ded1e7cb9ef7a8e401e71c9accd6133a68b5/data/map%20planets.txt#L4531-L4536
Especially when it's in the same system as Martini, which is clearly the bigger tourist attraction.

And Hermes:
https://github.com/endless-sky/endless-sky/blob/02e6ded1e7cb9ef7a8e401e71c9accd6133a68b5/data/map%20planets.txt#L2120-L2125

Also, what ordering is in use here?

Reply:
Removed those planets from the list. I alphabetized the list.